si.tano.Vremenar.yaml

```diff
@@ -0,0 +1,59 @@
+---
+app-id: si.tano.Vremenar
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vremenar
+rename-icon: vremenar
+rename-desktop-file: vremenar.desktop
+copy-icon: true
+finish-args:
+  # Required due to being a GUI application
+  - "--socket=x11"
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - "--share=ipc"
+  # Required for experimental wayland support
+  - "--socket=wayland"
+  # Required for OpenGL
+  - "--device=dri"
+  # As a weather application, networking is required
+  - "--share=network"
+  # Required for notifications in various desktop environments
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  # Required due to KDE's special way of handling notifications
+  - "--own-name=org.kde.StatusNotifierItem-2-1"
+  # Required for positioning
+  - "--system-talk-name=org.freedesktop.GeoClue2"
+cleanup:
+  - '/lib/liba*'
+  - '/lib/libb*'
+  - '/lib/libc*'
+  - '/lib/libd*'
+  - '/lib/libe*'
+  - '/lib/libgc*'
+  - '/lib/libgt*'
+  - '/lib/libl*'
+  - '/lib/libp*'
+  - '/lib/libs*'
+  - '/lib/libx*'
+  - '/lib/libX*'
+modules:
+  - name: vremenar
+    build-options:
+      strip: false
+      no-debuginfo: true
+    buildsystem: simple
+    build-commands:
+      - cp -r usr/* /app
+      - install -Dm644 si.tano.Vremenar.appdata.xml /app/share/appdata/si.tano.Vremenar.appdata.xml
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/ntadej/Vremenar/releases/download/v0.3.1/Vremenar_0.3.1_x86_64.tar.bz2
+        sha256: 6f4dbd222d54e80fdde98fa2f167d0e90ce83b26f5b2c931a0fd23b8daef1d2f
+      - type: file
+        path: si.tano.Vremenar.appdata.xml
```

Comment:
will this be upstream in a future release?

Reply:
Probably. I'm new to this type of metadata so for now I did not include it upstream yet.